sceptre/resolvers/s3.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from sceptre.resolvers import Resolver
+
+
+class S3(Resolver):
+    """
+    Resolver for the contents of an object in an S3 bucket.
```

Comment:
Is that really the case? I think it will just hook into what Sceptre already does by default after downloading, which means that if you specify a `template_bucket_name` in the StackGroup config, the downloaded file will get uploaded to an S3 bucket again and the filesize limit does not apply. The reason I set it up this way was to be able to hook into all existing Sceptre features, because to Sceptre, the downloaded file is just a template like any other template on disk.

Reply:
actually you are right, I forgot that sceptre uploads templates to the `template_bucket_name`. Please ignore my previous comment.